ClientApp/src/components/pages/DashboardPage/widgets/LoginTrackingWidget/LoginTrackingWidget.scss

```
@@ -0,0 +1,4 @@
+.login-widget-body {
+  max-height: 50vh;
+  overflow-y: auto;
+}
```

```suggestion
.login-widget-body {
  max-height: 50vh;
  overflow-y: auto;
  margin-right: 18px;
}
```
You can add this to show the scroll bar. It's currently being hidden by the white border.